sdk/metricsadvisor/Azure.AI.MetricsAdvisor/developer.md

```diff
@@ -0,0 +1,92 @@
+## Metrics Advisor Grow Up Story Experiment
+
+This is an experiment project which describes Data plane Codegen(DPG) grow up story, it uses Metrics Advisor as an example. For this experiment, we follow the steps below:
+- Generate DPG of Metrics Advisor.
+- Grow the SDK to feature equivalence to the current SDK. 
+- To confirm feature equivalence, run the existing live tests.
+
+In this experiment, we have updated configurations in autorest.md, manually added customized codes such as models, customized authentication, convenient methods to convert input model to RequestContent and Response to output model. In this tutorial, we will explain the step by step process of this experiment.
+
+### 1. Update autorest.md
+
+We have updated the following configurations in the autorest.md:
+* In order to generate DPG, we have added `data-plane: true` flag.
+* For AAD authentication, we have defined `security` and `security-scopes` configurations.
+* There are two clients, i.e. `MetricsAdvisorAdministrationClient` and `MetricsAdvisorClient` in the released Metrics Advisor library but Swagger spec doesn't define operation group in `operationId`, so only one client `MetricsAdvisorClient` is generated. In order to generate `MetricsAdvisorAdministrationClient` client, we have added [Generate MetricsAdvisorAdministrationClient](https://github.com/ShivangiReja/azure-sdk-for-net/blob/MetricsAdvisor-Experiment/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/autorest.md#generate-metricsadvisoradministrationclient) configuration.
+
+For more details, please refer to [`autorest.md`](https://github.com/ShivangiReja/azure-sdk-for-net/blob/MetricsAdvisor-Experiment/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/autorest.md) for the configuration.
+
+### 2. Gaps between generated code and released code
+
+In order to add grow up methods on the top of DPG generated code, we first need to see the gaps between generated code and [released code](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src). The gaps are:
+
+* Models and Headers are not generated in the generated folder.
+* In released library, `MetricsAdvisorAdministrationClient` is in the `Azure.AI.MetricsAdvisor.Administration` namespace. So we have added customization [here](https://github.com/ShivangiReja/azure-sdk-for-net/blob/MetricsAdvisor-Experiment/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs#L20) to move `MetricsAdvisorAdministrationClient` into `Azure.AI.MetricsAdvisor.Administration` namespace.
```

Comment:
What is the detail of this customization, just changing the namespace of MetricsAdvisorAdministrationClient?